src/tracing/errors.ts

```diff
@@ -0,0 +1,23 @@
+import { ErrorEvent, EventHint } from '@sentry/types'
+
+/** Identifies ethers request errors (as thrown by {@type import(@ethersproject/web).fetchJson}). */
+function isEthersRequestError(error: Error): error is Error & { requestBody: string } {
+  return 'requestBody' in error && typeof (error as unknown as Record<'requestBody', unknown>).requestBody === 'string'
+}
+
+export function filterKnownErrors(event: ErrorEvent, hint: EventHint): ErrorEvent | null {
```

Comment:
nit: we should document that returning `null` is how you prevent the error from being logged. i couldn't find this explicitly mentioned in the docs, but if you can then maybe we should link to it.

Reply:
It's documented through the type definition of beforeSend. I can enforce that `filterKnownErrors` matches that typedef!